### ISSUE TITLE
Task-by-Task Breakdown

### ISSUE THREAD

**Issue by perixRoshan** [perixRoshan/crop_pest_detection#1]
<html>
<body>

July 6
On July 6, I created the GitHub repository and project board to manage my work.

</body>
</html>
<html>
<body>

July 7
On July 7, I researched pest detection models like YOLOv8 and MobileNet, and finalized YOLOv8 for its efficiency.

</body>
</html>
<html>
<body>

July 8
--<html>
<body>

 
--

</body>
</html>

</body>
</html>

<html>
<body>

July 9
--On July 9, I cleaned and resized the images to prepare them for training.

</body>
</html>

<html>
<body>

July 10
--On July 10, I annotated the images using Roboflow’s labeling tool.

</body>
</html>
<html>
<body>

July 11
--On July 11, I trained the YOLOv8 model using the annotated dataset.

</body>
</html>

<html>
<body>

July 12
--<html>
<body>

 
--On July 12, I evaluated the model using precision, recall, and mAP scores.

</body>
</html>

</body>
</html>
<html>
<body>

July 13
--<html>
<body>

 
--On July 13, I fine-tuned the model by adjusting hyperparameters like learning rate and batch size.

</body>
</html>

</body>
</html>
<html>
<body>

July 14
--On July 14, I developed a frontend using Streamlit to allow users to upload pest images.

</body>
</html>

<html>
<body>

July 15
--<html>
<body>

 
--<html>
<body>

 
--

</body>
</html>

</body>
</html>

</body>
</html>
